README.md

```diff
@@ -10,13 +10,21 @@ A stylesheet(css) generator for use with Discord.
 
 |  | Testing |  |
 |--|--|--|
-|  | 		-Client + Firefox-		 |  |
-|  |-  (Tests have been done with the client and Firefox)  |  
+|  | 		All tested		 |  |
+|  |-  (Tests have been done with the client and Browsers)  |  
 ***
  - [x] Client (BD(BetterBiscord)~(100% Working!))
- - [ ] Client (Other css injectors)
  - [x] Firefox Browser (Testing done with stylish(DO NOT USE(Considered spyware)) 
```

Comment:
Suggest [Stylus](https://addons.mozilla.org/en-US/firefox/addon/styl-us/) for both Chrome and Firefox.

Reply:
Alright, done. resolving

Reply:
wtf

i told you to not recommend stylish because it's spyware, and i also told you about stylus... what did you do? Keep stylish mentioned but add "do not use cuz spyware"

Reply:
also there are 3 opening brackets, but only 2 closing brackets.

Reply:
I was not recommending its usages, just saying that I did use it for a test, but yes, I do understand, it's spyware, so I have edited to fix such